### PR TITLE
Adding schemas usage to the theme chooser sample.

### LINKED ISSUE #855: Add schemas to theme chooser sample.
Because of this [issue](https://github.com/IgniteUI/igniteui-docfx/issues/876) we should provide a sample with schemas usage

### PR DESCRIPTION
**Pull request by HristoP96**
Closes #855

### CODE REVIEW

File: src/app/theming/styles/themes.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // Import the theme utilities first
 @import '~igniteui-angular/lib/core/styles/themes/utilities';
+@import '~igniteui-angular/lib/core/styles/themes/schemas/_index';
```
Comment (simeonoff) [inline src/app/theming/styles/themes.scss]: No need to import this as it's already imported via utilities.

Comment (HristoP96) [reply src/app/theming/styles/themes.scss]: The redundant import is removed

File: src/app/theming/styles/schemas.scss [hunk 2]
```diff
@@ -0,0 +1,188 @@
+@import '~igniteui-angular/lib/core/styles/themes/utilities';
+
+$black-color: #000000;
+$aqua-color: #00BFFF;
+
+$black-theme-palette: igx-palette($primary: $black-color, $secondary: $aqua-color);
```
Comment (simeonoff) [inline src/app/theming/styles/schemas.scss]: Ensure you set the $grays property to #fff for dark palettes.

Comment (HristoP96) [reply src/app/theming/styles/schemas.scss]: $grays added to the palette

**Pull request merged**